ui/nuxeo-data-table/data-table-cell.js

```diff
@@ -19,6 +19,7 @@ import './data-table-templatizer-behavior.js';
             align-items: center;
             overflow: hidden;
             transition: flex-basis 200ms, flex-grow 200ms;
+            @apply --nuxeo-data-table-cell;
```

Comment:
To keep the naming consistent with other mixins (like [https://github.com/nuxeo/nuxeo-elements/blob/improve-ELEMENTS-1316-enhance-nuxeo-data-table-styling/ui/nuxeo-data-table/iron-data-table.js#L128](data table), I think this should be something like: 

```
@apply --iron-data-table-cell;
```
or 

```
@apply --data-table-cell;
```

Reply:
I believe `--iron-data-table-cell` is the best option here, for consistency with the other variables.